pkgs/top-level/all-packages.nix

```diff
@@ -11586,6 +11586,7 @@ with pkgs;
     inherit (python2Packages) python gyp;
     cctools = darwin.cctools;
     icu = icu58; # v8-5.4.232 fails against icu4c-59.1
+    stdenv = overrideCC stdenv gcc6;
```

Comment:
Can you alter the compiler only on linux and not on `darwin` at least?

Reply:
sure, will do